install_config/aggregate_logging.adoc

```diff
@@ -753,7 +760,7 @@ xref:../install_config/install/advanced_install.adoc#configuring-node-host-label
 [[fluentd-use-journald]]
 *Having Fluentd Use the Systemd Journal as the Log Source*
 
-By default, Fluentd reads from *_/var/log/messages_* and
+By default, Fluentd reads from `journald` and
```

Comment:
Fluentd will always use the `journald` as the system log source.  These are log messages from the operating system, from docker, and from openshift.  For container logs, Fluentd will determine which log driver docker is using - json-file or journald - and will automatically read the logs from that source.